src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx

```diff
@@ -136,11 +137,58 @@ class DataDownload extends AsyncView<Props, State> {
       </React.Fragment>
     );
   }
+
+  openInDiscover() {
+    const {
+      download: {
+        query: {info},
+      },
+    } = this.state;
+    const {orgId} = this.props.params;
+
+    const to = {
+      pathname: `/organizations/${orgId}/discover/results/`,
+      query: info,
+    };
+
+    browserHistory.push(to);
+  }
+
+  renderOpenInDiscover() {
+    const {
+      download: {
+        query = {
+          type: ExportQueryType.IssuesByTag,
+          info: {},
+        },
+      },
+    } = this.state;
+
+    // default to IssuesByTag because we dont want to
+    // display this unless we're sure its a discover query
+    const {type = ExportQueryType.IssuesByTag} = query;
+
+    return type === 'Discover' ? (
+      <React.Fragment>
+        <p>{t('Need to make changes?')}</p>
```

Comment:
It would be good to have some margin between this button and the checksum. Right now they are really close and look like one block.

Reply:
I've updated the screenshot